test/Query.test.ts

```diff
@@ -0,0 +1,69 @@
+import { gql } from "graphql-tag";
+import { ApolloServer } from "apollo-server-express";
+import { resolvers } from "../src/resolvers";
+import fs from "fs";
+const { expect } = require("chai");
+
+const typeDefs = fs.readFileSync("./src/schema/schema.graphql", "utf8");
+
+it("Query Example Test", async () => {
+	const testServer = new ApolloServer({
+		typeDefs,
+		resolvers,
+	});
+
+	let result = await testServer.executeOperation({
```

Comment:
Looks good. I'd make this `const` since you're not going to be changing it once each test is in its own block.